automation/jenkins_build.sh

```diff
@@ -126,6 +126,24 @@ cp $DEVICE_TYPE_JSON $BUILD_DEPLOY_DIR/device-type.json
 # move to deploy directory the kernel modules headers so we have it as a build artifact in jenkins
 mv -v $WORKSPACE/build/tmp/deploy/images/$MACHINE/kernel_modules_headers.tar.bz2 $BUILD_DEPLOY_DIR
 
+# If this is a clean production build, push a resinhup package to dockerhub
+if [[ "$sourceBranch" == production* ]] && [ "$metaResinBranch" == "__ignore__" ] && [ "$supervisorTag" == "__ignore__" ]; then
+    echo "INFO: Pushing resinhup package to dockerhub."
+    SLUG=$(jq --raw-output '.slug' $DEVICE_TYPE_JSON)
+    DOCKER_REPO="resin/resinos"
+    DOCKER_TAG="resinhup-$SLUG-$VERSION_HOSTOS"
```

Comment:
ok, let's change it there too. I don't think we'll ever have a non-resinhup ResinOS docker image. We're pushing towards unification